bindings/java/README.md

```diff
@@ -4,14 +4,11 @@
 
 ### 1. Build & install shared library
 - Follow [build.md](https://github.com/WasmEdge/WasmEdge/blob/master/docs/build.md) to build and install WasmEdge
```

Comment:
This is a dead link, should be https://wasmedge.org/book/en/contribute/build_from_src.html now

Reply:
Thanks for your review, fixed them.